### PR TITLE
Ada2 24

### PR DESCRIPTION
**Pull request by elzacontiero**
Branch ADA2-24 is ready for merge to master and already contains changes for the frontend from branch ADA2-23.

### CODE REVIEW

**Review by tim-barton96 — APPROVED** [review]
Looks all good and worked for me

File: AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/models/ReportApiModelBase.java [hunk 1]
```diff
@@ -7,6 +7,9 @@ public class ReportApiModelBase {
     private int reportId;
     private byte status;
     private ZonedDateTime reportTime;
+    private String reportTitle;
+
+
```
Comment (RogVan) [inline AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/models/ReportApiModelBase.java:12]: Probably best to not have any spaces here for readability

File: AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/models/ReportApiModelBase.java [hunk 2]
```diff
@@ -49,4 +60,12 @@ public int getAgentId() {
...
         this.agentId = agentId;
     }
+
+    @Override
+    public String toString() {
+        return "ReportApiModelBase [reportId=" + reportId + ", status=" + status + ", reportTime=" + reportTime
+                + ", reportTitle=" + reportTitle + ", reportBody=" + reportBody + ", agentId=" + agentId + "]";
+    }
+
+
```
Comment (RogVan) [inline AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/models/ReportApiModelBase.java:70]: Also don't need these extra spaces

File: AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/ReportBase.java [hunk 3]
```diff
@@ -34,6 +35,15 @@ public void setReportTime(LocalDateTime reportTime) {
...
 
+    public String getReportTitle() {
+        return reportTitle;
+    }
+
+    public void setReportTitle(String reportTitle) {
+        this.reportTitle = reportTitle;
+    }
+
+
```
Comment (RogVan) [inline AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/ReportBase.java:46]: Extra space

**Review by RogVan — COMMENTED** [review]
Sorry this is too late to change before you merge in, but just for feedback purposes it looks good to me.

A very minor point but try to avoid having too many extra blank lines separating out code, we would generally only use one blank line at most, just makes the code a bit harder to read if there are loads of gaps

**Pull request merged**